solutions/usaco-599.mdx

```diff
@@ -25,133 +25,169 @@ Bessie's location if the current path is not repeated somewhere else in the map.
 If we were to frame this as a string-matching problem, we would want to find the
 shortest substring of the map such that it only appears once.
 
+To handle angles, we can note that there are only two possible interior angles, $90^\circ$ and $270^\circ$.
```

Comment:
do we not have $\degree$ (`$\degree$`)?

Reply:
not sure. shouldn't really matter thougu